scripts/deploy.py

```diff
@@ -206,5 +232,18 @@ def _execute_deployment():
     print 'Done!'
 
 
+def update_cache_slug():
+    """Updates the cache slug in cache_slug.yaml"""
+    cache_slug = utils.get_unique_id()
```

Comment:
This file really should not depend on utils (or anything in oppia). Can you move get_unique_id() here instead?